### ISSUE TITLE
[Feature]: Add Llama 3.3 on Bedrock

### ISSUE THREAD

**Issue by steffen-sbt** [BerriAI/litellm#7329]
### The Feature

As of yesterday, Llama 3.3 is finally available on AWS Bedrock (https://aws.amazon.com/about-aws/whats-new/2024/12/metas-llama-3-3-70b-model-amazon-bedrock/), but not yet supported by LiteLLM.

Since I didn't see another issue for this, create one here. 

### Motivation, pitch

Llama 3.3 is excellent and cheap, and should be supported. 

### Are you a ML Ops Team?

Yes

**Comment by ghost** [comment]
Hey @steffen-sbt you can call models without waiting on us to onboard them by just specifying the bedrock/converse route - https://docs.litellm.ai/docs/providers/bedrock#set-converse--invoke-route

**Comment by steffen-sbt** [comment]
@krrishdholakia I don't think this is closed yet. I just updated to 1.55.9 which includes the changed from the branch above, but I still get 
```
{
      "aws_region_name": "us-east-1",
      "model": "bedrock/us.meta.llama3-3-70b-instruct-v1:0",
      "cache": {
        "no-cache": true
      },
      "max_tokens": 1,
      "error": "litellm.NotFoundError: BedrockException - Bedrock HTTPX: Unknown provider=us, model=us.meta.llama3-3-70b-instruct-v1:0\nHave you set 'mode' - https://docs.litellm.ai/docs/proxy/health#embedding-models\nstack trace: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.11/site-packages/litellm/main.py\", line 2539, in completion\n    response = bedrock_chat_completion.completion(\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/site-packages/litellm/llms/bedrock/chat/invoke_handler.py\", line 792, in completion\n    raise BedrockError(\nlitellm.llms.bedrock.common_utils.BedrockError: Bedrock HTTPX: Unknown provider=us, model=us.meta.llama3-3-70b-instruct-v1:0\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.11/site-packages/litellm/main.py\", line 5198, in ahealth_check\n    response = await ahealth_check_chat_models(\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/site-packages/litellm/main.py\", line 5060, in ahealth_check_chat_models\n    await acompletion(**model_params)\n  File \"/usr/local/lib/python3.11/site-packages/litellm/utils.py\","
    },
  ```
  when running the health check. Apologies, I should have added these details to the original issue. 
  
  It seems the cross-region inference includes some hardcoded list somewhere?
